### ISSUE TITLE
Accessibility: Add missing Focus rings

### ISSUE THREAD

**Issue by SabinaHrabinova** [bratislava/marianum.sk#364]
Zameranie interaktívnych prvkov nie je viditeľné.

Please add outline ("focus ring") when interactive elements are keyboard focused.
Outline is important for accessibility. Now we do not display it at all.

Best implementation in olo.sk
